spec/services_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+# require "spec_helper"
+require "services"
+
+RSpec.describe Services do
+  describe "#cache_timestamp" do
+    it "is set and recent" do
```

Comment:
This is probably fine? We could maybe use https://github.com/travisjeffery/timecop if need be but I'm not sure exactly what we'd do with it.